### ISSUE TITLE
Workarounds in M2 web application

### ISSUE THREAD

**Issue by Divikarur** [mifos-x-customization/os#55]
1.	Char- make default – positive
2.	Mobile no.- as of now, it will allow to enter more than 10, on submitting it will throw err. But now, we want like it should not even allow us to enter more than 10
3.	Client type – make default – salaried
4.	Cleint classification – make default – daily income
5.	In create client page – make staff mandatory
6.	Address type – default – home
7.	State/country – default – Tamil Nadu/ India
8.	Postal code – it should allow only 6 digits to enter..
9.	Role – default – member
10.	Make role mandatory
11.	Role update not working
12.	Under checker Inbox/tasks -> loan approval /disbursal – include center/grp field for clients displayed
13.	If we have not attach meeting in a center, it should allow to submit bulk loan JLG
14.	In Dem vs Collection , add a column to display day (Monday) based on repayment date / we can pick it from the attach meeting table where we will store the day.
15.	Any of these three must be entered – aadhar/voter/ration
16.	Bulk import – center/grp/client/loan/repayments
17.	Remove the field len validation on voter/ration

**Comment by maektwain** [comment]
> * Char- make default – positive
> * Mobile no.- as of now, it will allow to enter more than 10, on submitting it will throw err. But now, we want like it should not even allow us to enter more than 10
> * Client type – make default – salaried
> * Cleint classification – make default – daily income
> * In create client page – make staff mandatory
> * Address type – default – home
> * State/country – default – Tamil Nadu/ India
> * Postal code – it should allow only 6 digits to enter..
> * Role – default – member
> * Make role mandatory
> * Role update not working
> * Under checker Inbox/tasks -> loan approval /disbursal – include center/grp field for clients displayed
> * If we have not attach meeting in a center, it should allow to submit bulk loan JLG
> * In Dem vs Collection , add a column to display day (Monday) based on repayment date / we can pick it from the attach meeting table where we will store the day.
> * Any of these three must be entered – aadhar/voter/ration

> * Bulk import – center/grp/client/loan/repayments
> * Remove the field len validation on voter/ration

Bulk import needs to be tested with extra fields we have added.
Rest of the things are done 

please confirm @Divikarur

**Comment by Divikarur** [comment]
Yes, i will create new tasks on BulkImport, we shall close this workarounds as rest of the things are completed.
